podvm/Dockerfile.podvm_builder

```diff
@@ -63,7 +63,15 @@ RUN echo $CAA_SRC
 
 RUN echo $CAA_SRC_REF
 
-RUN git clone ${CAA_SRC} -b ${CAA_SRC_REF} cloud-api-adaptor
+RUN if [ -n "${CAA_SRC}" ]; then \
+      rm -rf cloud-api-adaptor && \
```

Comment:
Hmm, is is likely the `cloud-api-adaptor` is present already? If so dropping everything via `rm -rf` seems like a waste to me as many objects can be reused. How about `git remote set-url origin ${CAA_SRC}`?

Reply:
Reset the remote url (and perhaps fetch the branch again?) seems a good idea. There a couple of places where we delete the repository and clone again, where we can use that idea. However, I would prefer to send in a separate PR as this change will need to re-test this PR (which is not an easy task :( Is it okay for you?

Reply:
Sure, it's just an optimization